buildSrc/src/main/java/com/soundcloud/reflect/DelectExtension.kt

```diff
@@ -2,6 +2,12 @@ package com.soundcloud.reflect
 
 open class DelectExtension {
     var daggerReflectVersion = "0.3.0"
+
+    /**
+     * Whether or not the plugin is enabled. If set, takes precedence over `gradle.properties` configuration
+     */
+    var enabled: Boolean? = null
```

Comment:
this looks great. can you please make this a gradle property?

Reply:
Yep, forgot the Groovy compat, sorry. Done :)